modules/ingress/README.md

```diff
@@ -1,61 +1,39 @@
+Besides, there may be multiple ingresses each of which can create an ALB, but this is expensive.
 
-```terraform
-module "ingress" {
-  source   = "dasmeta/modules/aws//modules/ingress"
-
-  alb_name = "test"
-  hostname = "test3.devops.dasmeta.com"
+## Solution
+We recommend to use this Terraform module which
+1. creates a k8s Ingress resource,
+2. created resource makes an ALB in AWS,
+3. has `alb.ingress.kubernetes.io/group.name` annotation set by default.
 
-  annotations = {
-    "alb.ingress.kubernetes.io/certificate-arn"    = "arn:aws:acm:us-east-1:5********68:certificate/a55ee6eb****1706"
-  }
-}
-```
+In the result of this you can
+1. manage the state of the resource by Terraform,
+2. by setting `alb.ingress.kubernetes.io/group.name` annotation attach multiple ingresses to the main one created by the module and use just one ALB,
+3. manage all the Ingress rules from the application side.
 
-# Example 2. create ingress for apiVersion: extensions/v1beta1
+# Example. Create ingress with an existing certificate, host and custom values for ingress annotations
 
 ```terraform
-module "ingress" {
-  source   = "dasmeta/modules/aws//modules/ingress"
+data "aws_acm_certificate" "issued" {
+  domain   = "dasmeta.com"
+  statuses = ["ISSUED"]
+}
 
-  alb_name = "test"
-  hostname = "test3.devops.dasmeta.com"
-  api_version = extensions/v1beta1
+module "ingress" {
+  source = "dasmeta/modules/aws//modules/ingress"
 
-  annotations = {
-    "alb.ingress.kubernetes.io/certificate-arn"    = "arn:aws:acm:us-east-1:5********68:certificate/a55ee6eb****1706"
-  }
-}
-```
+  name     = "terraform-ingress
+  hostname = "dasmeta.com"
 
-# Example 3. Create ingress override annotations
+  certificate_arn           = data.aws_acm_certificate.issued.arn
+  healthcheck_path          = "/health"
+  healthcheck_success_codes = "200-399"
 
-```terraform
-module "ingress" {
-  source   = "dasmeta/modules/aws//modules/ingress"
-
-  alb_name = "test"
-  hostname = "test.devops.dasmeta.com"
-  annotations = {
-       "alb.ingress.kubernetes.io/load-balancer-name" = "test-ingress"
-       "alb.ingress.kubernetes.io/scheme"             = "internet-facing"
-       "alb.ingress.kubernetes.io/subnets"            = "subnet-0ebc13842a5f, subnet-05b72a3769b, subnet-0da2ece4bb4229"
-       "alb.ingress.kubernetes.io/backend-protocol"   = "HTTP"
-       "alb.ingress.kubernetes.io/certificate-arn"    = "arn:aws:acm:us-east-1:5********68:certificate/a55ee6eb****1706"
-       "alb.ingress.kubernetes.io/group.name"         = "dev"
-       "alb.ingress.kubernetes.io/listen-ports"       = "[{\"HTTPS\":443}, {\"HTTPS\":80}]"
-       "kubernetes.io/ingress.class"                  = "alb"
-     }
-  path = [
-    {
-      service_name = "nginx"
-      service_port = "80"
-      path         = "/alb-terraform-created"
-    }
-  ]
+  tls_hosts = "dasmeta.com"
```

Comment:
Don't we want to give working example to the user? I mean ingress manifest that can link with the main ingress by group. We do have it in examples folder but better have it here to have it complete.